src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/messenger_transport_serializer_config.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd
+        http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config>
+        <framework:messenger>
+            <framework:serializer default-serializer="messenger.transport.symfony_serializer">
+                <framework:symfony-serializer format="xml">
+                    <framework:context>
+                        <framework:some_context>true</framework:some_context>
+                    </framework:context>
+                </framework:symfony-serializer>
+            </framework:serializer>
+            <framework:transport name="foo" dsn="null://">
+                <framework:symfony_serializer format="xml">
+                    <framework:context>
+                        <framework:some_other_context>true</framework:some_other_context>
```

Comment:
tests are failing here
```
1) Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\XmlFrameworkExtensionTest::testMessengerWithTransportSerializerConfig
Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: Unable to parse file "/home/nikophil/works/symfony/src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/messenger_transport_serializer_config.xml": [ERROR 1871] Element '{http://symfony.com/schema/dic/symfony}symfony_serializer': This element is not expected. Expected is one of ( {http://symfony.com/schema/dic/symfony}options, {http://symfony.com/schema/dic/symfony}retry-strategy ). (in /home/nikophil/works/symfony/ - line 18, column 0)
```
don't know how to make them pass

Reply:
The schema needs an update
https://github.com/symfony/symfony/blob/6b9fafb2c0ef84bfa7d17b3b37737c5da710bd0e/src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd#L499

Reply:
ok thanks, I was thinking it was automatically generated